adds a basic testimonial pattern

aplauche opened the pull request:
Current pattern uses wide alignment, could swap to default to normal width if that makes more sense.

aplauche committed:
adds a basic testimonial pattern

aplauche committed:
switch to xl font size

aplauche committed:
remove constrained layout

kishanjasani commented:
@carolinan Weather it is dark mode or light mode in styles variation. This patterns background color is Black and foreground is white. Check this link: https://prnt.sc/pOPADkpmakFO

This pattern uses `.has-tertiary-color` class [Check this code.](https://github.com/WordPress/twentytwentyfour/pull/36/files#diff-bb21203ac90c451af02b0e7cf83642398217283bc83799f195ec020c0090db27R11) which will convert background color to white and text color to black in `Dark Serif  style variation`. What could be the best solution for this. Introduce a new color variable in theme.json?

carolinan commented:
I recommend keeping it as is without adding more colors.

MaggieCabrera commented on patterns/testimonial.php:
```diff
@@ -0,0 +1,27 @@
+?>
+
+<!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"140px","bottom":"140px"}}},"backgroundColor":"contrast","textColor":"tertiary","layout":{"type":"default"}} -->
+<div class="wp-block-group alignwide has-tertiary-color has-contrast-background-color has-text-color has-background" style="padding-top:140px;padding-bottom:140px"><!-- wp:paragraph {"align":"center","style":{"spacing":{"margin":{"bottom":"32px"}},"typography":{"lineHeight":"1.2"}},"textColor":"tertiary","fontSize":"x-large","fontFamily":"cardo"} -->
```
Let's use rems instead of px for the margins and paddings all over the pattern

aplauche replied:
Will do! Thanks

MaggieCabrera commented on patterns/testimonial.php:
```diff
@@ -0,0 +1,27 @@
+
+<!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"140px","bottom":"140px"}}},"backgroundColor":"contrast","textColor":"tertiary","layout":{"type":"default"}} -->
+<div class="wp-block-group alignwide has-tertiary-color has-contrast-background-color has-text-color has-background" style="padding-top:140px;padding-bottom:140px"><!-- wp:paragraph {"align":"center","style":{"spacing":{"margin":{"bottom":"32px"}},"typography":{"lineHeight":"1.2"}},"textColor":"tertiary","fontSize":"x-large","fontFamily":"cardo"} -->
+  <p class="has-text-align-center has-tertiary-color has-text-color has-cardo-font-family has-x-large-font-size" style="margin-bottom:32px;line-height:1.2"><em><?php echo esc_html__('“Études has saved us thousands of hours of work and has unlocked insights we never thought possible.”', 'twentytwentyfour'); ?></em></p>
```
The content of the paragraph should be separated from the borders of its container. It's hard to see in figma, but the horizontal spacing should be of 90px (5.625rem) on each side

MaggieCabrera reviewed (changes requested):
Thank you for your PR! just a few comments on it and it should be good to go

MaggieCabrera commented:
If you rebase the PR agains trunk you will get the latest changes for the color palette. This pattern should be using Contrast as a background color and base as a text color

aplauche committed:
updates padding and margin to rems, adds side padding, and uses base color

aplauche committed:
adds spaces and newline to address linting errors

aplauche committed:
undo extra space in function call

aplauche committed:
clean up pattern spacing line breaks

aplauche commented:
Hmm, added changes from review, but currently getting a generic "Can't use 'tar -xzf' extract archive file" on code checks? Was working earlier and caught a few formatting things that I fixed. Can look more into it a little later today.
